### ISSUE TITLE
Fix alert "Transitdata prod swarm VM available disk space"

### ISSUE THREAD

**Issue by teemu8655** [HSLdevcom/transitdata#282]
At the moment alert "Transitdata prod swarm VM available disk space" is not directed to Solita webhook, because alert does not funcion correctly. When active, alert is triggered every 5 minutes and does not resolve correctly.

Fix alert so, that it will appear only once when triggered, and fix also that it will resolve correctly.

After the fix, update terraform and make sure that the alert is directed to Solita prod (Common solita alerts prod)

**Comment by mjaakko** [comment]
This should make the alert resolve correctly: https://dev.azure.com/hslfi/Azure-automation/_git/azure-infra-transitdata-alerts/pullrequest/7436

**Comment by mjaakko** [comment]
Change deployed to prod, open a new issue if it does not work as intended
